tests/containers/podmansh.pm

```diff
@@ -0,0 +1,200 @@
+# SUSE's openQA tests
+#
+# Copyright 2021-2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test rootless mode on podman.
+# - add a user on the /etc/subuid and /etc/subgid to allow automatically allocation subuid and subgid ranges.
+# - check uids allocated to user (inside the container are mapped on the host)
+# - give read access to the SUSE Customer Center credentials to call zypper from in the container.
+#   This grants the current user the required access rights
+# - Test rootless container:
+#   * container is launched with default root user
+#   * container is launched with existing user id
+#   * container is launched with keep-id of the user who run the container
+# - Restore /etc/zypp/credentials.d/ credentials
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use Utils::Systemd qw(systemctl);
+use utils;
+use containers::common;
+
+my $quadlet_dir = '/etc/containers/systemd';
+
+my $src_image = "registry.opensuse.org/opensuse/leap:15.6";
+my $quadlet_container = <<_EOF_;
+[Unit]
+Description=The podmansh container
+After=local-fs.target
+
+[Container]
+Image=$src_image
```

Comment:
I think this can be made static here and perhaps it's a better approach to move this file to `data/containers/` to avoid mixing test code with test data?

Reply:
Can't. I'm using the image tag to pre-pull the image down in the code (unless we don't care about duplicates).